src/app/routes/getInitialData/index.js

```diff
@@ -6,19 +6,48 @@ import onClient from '#lib/utilities/onClient';
 import getBaseUrl from './utils/getBaseUrl';
 import getPreprocessorRules from './utils/getPreprocessorRules';
 
-const logger = nodeLogger(__filename);
+export const logger = nodeLogger(__filename);
```

Comment:
If this is for mocking, we have this: https://github.com/bbc/simorgh/blob/41205ae24deb481e7bfaeb190651188602d631ac/src/testHelpers/loggerMock.js